### PR TITLE
Prog report: better line plot colors and background

### LINKED ISSUE #1765: improve colors in prognostic run report
not colorblind friendly and yellow is hard to see. Here is an example of a good colorscheme: https://gist.github.com/nbren12/585e73cb5543568bc83f220f85890870

No bokeh example though.

could also change the background color to improve contrast.

### PR DESCRIPTION
**Pull request by brianhenn**
Updates the line plots in the prognostic run report to use a color cycle with better visibility as well as gray backgrounds with white grids, and makes black lines for verification. Sources this common color cycle for usage in fv3viz, the prognostic run report, and the logs viewer instead of defining separate cycles in each. 

See [before](https://storage.googleapis.com/vcm-ml-public/argo/2022-04-11t231627-e8c1d74aa907/index.html) and [after](https://storage.googleapis.com/vcm-ml-public/brianh/test-prog-report/report/index.html). 

Resolves #1765.

### CODE REVIEW

File: workflows/diagnostics/fv3net/diagnostics/prognostic_run/views/static_report.py [hunk 1]
```diff
@@ -157,7 +157,12 @@ def _set_opts_and_overlay(hmap, overlay="run"):
     return (
         hmap.opts(norm={"framewise": True}, plot=dict(width=850, height=500))
         .overlay(overlay)
-        .opts(legend_position="right")
+        .opts(
+            legend_position="right",
+            bgcolor="lightgray",
```
Comment (oliverwm1) [inline workflows/diagnostics/fv3net/diagnostics/prognostic_run/views/static_report.py:162]: nit: to me the plot would be a little clearer if the background was lighter. Is there something between lightgray and white? Obviously pretty subjective...

Comment (brianhenn) [reply workflows/diagnostics/fv3net/diagnostics/prognostic_run/views/static_report.py:165]: Apparently 'whitesmoke' is a color.

File: workflows/diagnostics/fv3net/diagnostics/prognostic_run/views/styles.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# https://davidmathlogic.com/colorblind
+modified_wong_palette = [
```
[the remark this answers is not in the document]
Comment (brianhenn) [reply workflows/diagnostics/fv3net/diagnostics/prognostic_run/views/styles.py]: Palette sourced from fv3viz for these applications as requested.

**Review by oliverwm1 — APPROVED** [review]
Thanks, I think the plots are quite a bit improved! The grid helps distinguish differences a bit more quantitatively.

Consider consolidating the `run == 'verification'` logic in one place. It's simple, but we might want to change implementation sometime, would be nice to just do it in one spot.

File: workflows/diagnostics/fv3net/diagnostics/prognostic_run/views/static_report.py [hunk 3]
```diff
@@ -94,16 +96,17 @@ def make_plots(self, data) -> Iterable:
...
     All matching diagnostics must be 1D."""
-    p = hv.Cycle("Colorblind")
+
     hmap = hv.HoloMap(kdims=["variable", "run"])
     vars_to_plot = run_diags.matching_variables(varfilter)
     for run in run_diags.runs:
         for varname in vars_to_plot:
             v = run_diags.get_variable(run, varname).rename("value")
             style = "solid" if run_diags.is_baseline(run) else "dashed"
+            color = "black" if run == "verification" else COLOR_CYCLE
```
[the remark this answers is not in the document]
Comment (brianhenn) [reply workflows/diagnostics/fv3net/diagnostics/prognostic_run/views/static_report.py:106]: Sure

**Pull request merged**